helpers/update-site/documentation.js

```diff
@@ -437,16 +469,29 @@ const getHintFileInfo = async (file) => {
      *
      * e.g.    this is a "description" => this is a \"description\"
      */
-    let description = metaDescription ? metaDescription[1].trim().replace(/\\`/g, '`')
-        .replace(/([^\\])"/g, '$1\\"') : (getPredefinedDescription(file.dest) || getDescription(file.content));
+    let description;
 
-    /*
-     * Remove the character the quotes from the text (' ` ") and the character ',' if it is the last character.
-     * e.g: 'Runs axe-core tests in the target', -> Runs axe-core tests in the target
-     */
-    if (meta) {
-        description = description[description.length - 1] === ',' ? description.substr(1, description.length - 3) : description.substr(1, description.length - 2);
+    if (metaDescription) {
+        description = metaDescription[1].trim().replace(/\\`/g, '`')
+            .replace(/([^\\])"/g, '$1\\"');
+
+        /*
+         * Remove the character the quotes from the text (' ` ") and the character ',' if it is the last character.
+         * e.g: 'Runs axe-core tests in the target', -> Runs axe-core tests in the target
+         */
+        if (!description.startsWith('getMessage')) {
+            description = description[description.length - 1] === ',' ? description.substr(1, description.length - 3) : description.substr(1, description.length - 2);
+        }
+    } else {
+        description = getPredefinedDescription(file.dest) || getDescription(file.content);
     }
+
+    if (description.startsWith('getMessage')) {
+        description = getDescriptionFromLocalization(file).trim()
+            .replace(/\\`/g, '`')
+            .replace(/([^\\])"/g, '$1\\"');
```

Comment:
Escaping `"` is potentially needed no matter where we get the string from, correct? If so, I'd prefer moving this (and other instances of it) to a single call after all conditional assignments to `description` are complete.